_posts/2023-10-25-aws-lambda-linode-migrate.md

```diff
@@ -0,0 +1,383 @@
+---
+title: "Migrate AWS Lambda functions to Linode Kubernetes Engine with OpenFaaS"
+description: "This guide will show you how to migrate existing AWS Lambda functions to OpenFaaS running on LKE"
+date: 2023-10-25
+categories:
+- openfaas
+- akamai
+- linode
+- lambda
+dark_background: true
+author_staff_member: Han
+image: "/images/2023-09-kafka-confluent/background.png"
+hide_header_image: true
+---
+
+In this guide we'll show you how to migrate existing AWS Lambda functions to a Akamai Cloud Computing LKE cluster with OpenFaaS.
+
+> Did you know? Linode was acquired by Akamai, and is now being branded as "Akamai Cloud Computing".
+> The rebranding is still in-progress, so we'll be referring to Linode throughout this article.
+
+This makes functions completely portable and allows running
+- Running at any scale with predictable cost.
+- Remove any limits and run workloads AWS Lambda is not suitable for with the 
+- Get a great developer experience build on familiar docker concepts.
+
+We will work through migrating a real-world extract, transform, and load (ETL) workflow from AWS to Linode with OpenFaaS.
+
+## Deploy OpenFaaS on Linode Kubernetes Engine
+
+See their [getting started guide](https://www.linode.com/docs/products/compute/kubernetes/get-started/) to setup an LKE cluster.
```

Comment:
Give the reader a few hints on what size of cluster you created, assuming that you did use LKE?

What GB RAM / vCPU and how many VMs should be used for this tutorial?